### ISSUE TITLE
Update podman-container-runlabel

### ISSUE THREAD

**Issue by Procyhon** [containers/podman#10799]
/kind bug

**Description**

While adapting the `podman-container-runlabel` manpage to the MANPAGE_SYNTAX there were some questions that came up since some things may not be explained enough.

The example command below shows the use of arguments as shown in the current manpage:
```
$ podman container runlabel install foobar apples oranges
```
What arguments can be put behind the `container image`?
I would assume that with the use of the `opt1/2/3` arguments a command would look something like this:
```
$ podman container runlabel run quay.io/baude/demodb:latest --opt1=test --opt2=test2 --opt3=test3
```
Is it also possible to put new environment variables behind the `container image` to extend the usability like in this example command (and if this is the case, which one comes first):
```
$ podman container runlabel run quay.io/baude/demodb:latest --opt1=test -e HOST=/host
```

Next up:
What are the `**LABEL**` and `**IMAGE**` sections meant to represent (Lines 26ff)? Are these variables like `opt1/2/3` too?
Are the `**SUDO_UID**` and `**SUDO_GID**` also there to represent additional variables?
Could someone explain the use of these, cause I don't really understand what is meant with `podman -u` in the explanation text?

**In summary:**
1. What is the synopsis for the arguments.
2. What arguments are supported?

I'd also like to suggest to explain all different use cases of the label in the only place where the label is created, in the `podman-build` manpage. I believe the `podman-container-runlabel` manpage should explain what it is doing with the label, but not what is in the label.

**Comment by Luap99** [comment]
@baude PTAL

**Comment by vrothberg** [comment]
I'll take a look
